syntax = "proto3";

package hyn;

message TranslateRequest
{
    string msg = 1;
};

message TranslateResponse 
{
    string msg = 1;
};

message Addrequest 
{
    int32 num1 = 1;
    int32 num2 = 2;
};

message AddResponse 
{
    int32 result = 1;
};